Resources/Prototypes/Entities/Objects/Devices/Circuitboards/Machine/production.yml

```
@@ -48,6 +48,20 @@
           DefaultPrototype: Beaker
           ExamineName: Glass Beaker
 
+- type: entity
+  id: AmmoTechFabCircuitboard
+  parent: BaseMachineCircuitboard
+  name: ammo techfab circuit board
+  description: A machine printed circuit board for a security techfab
+  components:
+    - type: Sprite
+      state: security
+    - type: MachineBoard
+      prototype: AmmoTechFab
+      requirements:
+        MatterBin: 3
+        Manipulator: 3
```

```suggestion
        MatterBin: 1
        Manipulator: 1
```
just a bit expensive for such a niche machine